DevoxxClientMobile/src/main/java/com/devoxx/views/helper/FilterSessionsPresenter.java

```diff
@@ -226,6 +226,18 @@ public void initialize() {
         updateSearchPredicate();
     }
 
+    public void hidePastSession() {
+        periodAll.setSelected(false);
+        for (Node child : periodRadioBtContainer.getChildren()) {
+            if (child.getUserData() == TimePeriod.MORE_THAN_ONE_HOUR_AGO) {
+                ((RadioButton) child).setSelected(true);
+                return;
+            }
+        }
+        selectedTimePeriod = TimePeriod.MORE_THAN_ONE_HOUR_AGO;
+        apply();
```

Comment:
Both of these steps looks mandatory to me. Should we replace `return` inside the loop with `break`?

```
public void hidePastSession() {
    periodAll.setSelected(false);
    for (Toggle toggle : radioButtonGroup.getToggles()) {
        if (toggle.getUserData() == TimePeriod.MORE_THAN_ONE_HOUR_AGO) {
            toggle.setSelected(true);
            break;
        }
    }
    selectedTimePeriod = TimePeriod.MORE_THAN_ONE_HOUR_AGO;
    apply();
}
```

Reply:
Good catch. (Copy-pasted resetPeriodUI)